cypress/e2e/firstTests.cy.js

```diff
@@ -0,0 +1,23 @@
+/// <reference types="cypress" />
+
+describe('', () => {
+  it('', () => {
+    cy.visit ('https://react-redux.realworld.io/#/login?_k=ks2hqr')
+
+    cy.get(':nth-child(1) > .form-control')
+    .type ('12345555@gmail.com')
```

Comment:
Methods should have the same indentation
```suggestion
      .type ('12345555@gmail.com')
```